src/plugins/event-plugins/WebVitalsPlugin.ts

```diff
@@ -1,12 +1,20 @@
 import { InternalPlugin } from '../InternalPlugin';
 import { LargestContentfulPaintEvent } from '../../events/largest-contentful-paint-event';
-import { FirstInputDelayEvent } from '../../events/first-input-delay-event';
 import { CumulativeLayoutShiftEvent } from '../../events/cumulative-layout-shift-event';
-import { Metric, onCLS, onFID, onLCP } from 'web-vitals';
+import { FirstInputDelayEvent } from '../../events/first-input-delay-event';
 import {
-    LCP_EVENT_TYPE,
+    CLSMetricWithAttribution,
+    FIDMetricWithAttribution,
+    LCPMetricWithAttribution,
+    Metric,
+    onCLS,
+    onFID,
+    onLCP
+} from 'web-vitals/attribution';
```

Comment:
question: By how much does using the web-vitals attribution build (+ adding our own code to record attribution) increase the total size of the compressed aws-rum-web package?

At some point we will also want to create different builds, of different sizes, for aws-rum-web.

Reply:
I did not do an exact comparison. There was no documentation on size difference in the web vitals package for different builds, so I just did my best to keep my changes as small as possible. 

I can check though.

Reply:
I used the [bundle analyzer plugin](https://github.com/webpack-contrib/webpack-bundle-analyzer) to do a size comparison. I compared parsed sizes of the `cwr.js`, `WebVitalsPlugin`, and `web-vitals` package. 

**release/2.0.x**
* total size = 212.76kb
* web vitals plugin = 1.24kb
* web-vitals package = 6.52kb

**this pr**
* total size = 215.82kb
* web vitals plugin = 9.56kb
* web-vitals package = 2.02kb

**diff**
* total size = + ~3kb
* web vitals plugins = + ~8kb
* web vitals package = -4kb?

Not sure why the attributions build is smaller than the normal build.